### ISSUE TITLE
Embed recent GH activity for a group

### ISSUE THREAD

**Issue by tripu** [w3c/Unitas#22]
Apparently a suggestion made during the AC in March 2016.

**Comment by tripu** [comment]
This is [where this might fit within the current prototype](https://w3c.github.io/Unitas/?g=68239#gh).

**Comment by tripu** [comment]
@guibbs, shall we tackle this one?

**Comment by guibbs** [comment]
yes! we'll use [pheme](https://github.com/w3c/pheme) to do that.

**Comment by tripu** [comment]
A comment by @guibbs yesterday (to clarify how we'll do this):

> &ldquo;We will use *Pheme* for realtime updates, such as comments, when an issue is open, commits, etc.; and [our own abstraction layer on top of the GH API] for general information such as user profile picts, etc.&rdquo;

**Comment by tripu** [comment]
https://github.com/w3c/ghapi ?
